k8s/demo/common/money-claims/cmc-demo.yaml

```diff
@@ -60,104 +65,31 @@ spec:
               secretRef: system-update-password
               key: key
         environment:
-          IDAM_API_URL: https://idam-api.demo.platform.hmcts.net
+          IDAM_API_URL: {{ .Values.global.idamApiUrl }}
```

Comment:
You don't need to override all these as they are all in values.yaml. this should simplify with latest changes